### ISSUE TITLE
Stuck at subdomain scanning

### ISSUE THREAD

**Issue by waelahmed-dev** [R0X4R/Garud#51]
This is my first time using the tool.
I just get stuck at subdomain scanning, I tried to leave it for about 3 hours and it was still stuck at the same phase, even I can't cancelling it by hitting `ctrl + c`, it won't cancel until I close my vps window
![169668922-e6b6ce86-204f-4aad-ae1a-32fe9554a293](https://user-images.githubusercontent.com/25941043/169669114-ea74e7cf-a8c6-45fe-b670-8333e8bdc54a.png)

Any ideas where is the problem?
Thanks in advance.

**Comment by R0X4R** [comment]
I think there's some issue with your amass try running your garud using and check where you get stuck and then let me know will fix it then.

Command: 

```css
$ bash -x /usr/bin/garud -d target.com
```

**Comment by waelahmed-dev** [comment]
this is the result:

![Screenshot_2022-05-22_11-27-30](https://user-images.githubusercontent.com/25941043/169689093-058adb01-26c2-4631-8c39-4b5322f291b4.png)

my vps specs:
2 GB RAM, 1 vCPU, 60 GB SSD
Node.js

I don't think that it has an env, it's amazon lightsail instance

**Comment by R0X4R** [comment]
Yes it requires higher specs to work faster but the specs you mentioned here is okay. Actually the issue is with the amass try commenting out the amass scans I'm pasting the line number and code below.

Run garud like this (use `screen` to let it run in background)

```css
$ env bash
$ garud -d target.com
```

Line 146
```css
# amass enum -passive -d $DM -o .tmp/amass.list &> /dev/null
```

Line 153
```css
# timeout 50m amass enum -active -brute -w ~/wordlists/subdomains.txt -d $DM -o .tmp/amassact.list &> /dev/null
```
